website/docs/docs/dbt-cloud-apis/sl-jdbc.md

```diff
@@ -403,6 +325,19 @@ select * from {{
                   }}
 ```
 
+### Query by all dimensions
```

Comment:
@courtneyholcomb added query by all dimensions from @Jstein77 [duplicate PR](https://github.com/dbt-labs/docs.getdbt.com/pull/6777/files#diff-a4e431c95f6eebc16186348ffe9d3245087cfb60303dfcf2af4b2f18d1d0176fR590-R599)

however should it be `semantic_layer.query_with_all_group_bys` or `semantic_layer.group_by_all()`? the example doesn't match the text above it?

Reply:
@mirnawong1 it's `semantic_layer.query_with_all_group_bys`

I'm wondering if this should be documented at all, though? This was an ask by Sigma, and we decided to put it into a separate method instead of just query because that's not really the intended way of using the Semantic Layer, and there are many caveats to it, including possibly getting incorrect data if you don't know what you're doing.

IMO regular users should just stick to `query()` and we shouldn't promise stability and/or backwards compatibility to `query_with_all_group_bys()` in the long run.

I wanna hear Courtney's and Jordan's thoughts on this!!

Reply:
sounds good, thank you! i copied it over from Jordan's pr https://github.com/dbt-labs/docs.getdbt.com/pull/6777/files#diff-a4e431c95f6eebc16186348ffe9d3245087cfb60303dfcf2af4b2f18d1d0176fR590-R599 so would be great to get clarity!

Reply:
@serramatutu I think we should document it! It's a useful endpoint that can simplify certain query requests for users. I also don't think there's much downside in putting documenting it.